packs/data/extinction-curse-bestiary.db/doblagub.json

```diff
@@ -112,7 +112,7 @@
             "type": "spellcastingEntry"
         },
         {
-            "_id": "v2ov1hPXLNCnQj1i",
+            "_id": "aR682CuMmY2ZuBvA",
```

Comment:
Comment actual applies to lines 73 to 76: should this have three level 6 spell slots, rather than 1?

Reply:
![image](https://user-images.githubusercontent.com/64166996/190524178-36b1e3c7-e605-4c77-9fa1-0d0609a04d41.png)

Since it is an innate spellcasting entry, it doesn't particularly matter. Those fields aren't used to determine how many spells fit in the level header.

Reply:
Doesn't hurt to change it anyway